engine/src/flutter/impeller/renderer/backend/vulkan/android/ahb_texture_source_vk.cc

```diff
@@ -8,9 +8,27 @@
 #include "impeller/renderer/backend/vulkan/context_vk.h"
 #include "impeller/renderer/backend/vulkan/texture_source_vk.h"
 #include "impeller/renderer/backend/vulkan/yuv_conversion_library_vk.h"
+#include "vulkan/vulkan_to_string.hpp"
 
 namespace impeller {
 
+namespace {
+
+static bool RequiresYCBCRConversion(vk::Format format) {
+  switch (format) {
+    case vk::Format::eG8B8R83Plane420Unorm:
```

Comment:
add `[[fallthrough]]`

https://google.github.io/styleguide/cppguide.html#Switch_Statements

Reply:
AFAIK from reading this we only need [[fallthrough]] if there is a body, not when we group multiple cases together?

Reply:
mebe :)